iron-overlay-behavior.html

```diff
@@ -115,7 +111,10 @@
       closingReason: {
         // was a getter before, but needs to be a property so other
         // behaviors can override this.
-        type: Object
+        type: Object,
+        value: {
```

Comment:
This needs to be...

```
value: function() {
  return { canceled: false };
}
```

Or else all instances will share the same default value instance.